### PR TITLE
ISSUE_524: Fix spelling and punctuation in some areas in 'Console Producer and Consumer Basics' page

### PR DESCRIPTION
**Pull request by tikimims**
- Fixed spelling and punctuation in some areas in **Console Producer and Consumer Basics, no (de)serializers** page. 

- https://github.com/confluentinc/kafka-tutorials/issues/524

### CODE REVIEW

File: _includes/tutorials/console-consumer-produer-basic/kafka/markup/dev/consume-topic-from-beginning.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Next let's open up a console consumer again, but this time you'll ready everthing that your producer has sent to the topic. you created in the previous step.
+Next let's open up a console consumer again, but this time you'll ready everything that your producer has sent to the topic. you created in the previous step.
```
Comment (tikimims) [inline _includes/tutorials/console-consumer-produer-basic/kafka/markup/dev/consume-topic-from-beginning.adoc:1]: @gAmUssA Hi Viktor, can you help with this one line here? I saw this one line and wanted to see what we were meaning to say in the line? Looks we're missing a word or two. I can update it here and I'll remove the extra "." as well.

Comment (gAmUssA) [reply _includes/tutorials/console-consumer-produer-basic/kafka/markup/dev/consume-topic-from-beginning.adoc:1]: Looks like . is indeed redundant but whole sentence is a pretty big and difficult to read. Feel free to split it.

**Comment by tikimims** [comment]
Hi @ybyzek,

This one is ready for your review - successful build. 👍

**Review by ybyzek — APPROVED** [review]
LGTM

**Pull request merged**